blaze-client/src/main/scala/org/http4s/blaze/client/Http1Connection.scala

```diff
@@ -310,105 +310,113 @@ private final class Http1Connection[F[_]](
         cb(Left(t))
     }(executionContext)
 
-  private def parsePrelude(
+  // it's called when the body is known
+  private def terminationConditionParsingPrelude(): Either[Throwable, Option[Chunk[Byte]]] =
```

Comment:
I would like to search for a better name for this method. It actually is relevant to parsing the body not the prelude. It's called when an EOF is received while parsing body, and the purpose of this metod it to decide whether it's ok that we recieved the EOF or not.

Maybe something like `onEofWhileReadingBody`?